docs/source/search-runs.rst

```diff
@@ -102,6 +102,7 @@ You can search using the following run attributes contained in :py:class:`mlflow
   # RHS value for start_time and end_time are unix timestamp
```

Comment:
Can the `Run Attributes` section above have `user_id` added in the 'string values' section?

Reply:
done 👍